Calling/BlobStorageHelperInfo.cs

```diff
@@ -7,14 +11,158 @@ public class BlobStorageHelperInfo
         public string Uri { set; get; }
     }
 
-    public struct RecordingInfo
+    public struct RecInfo
     {
-        public RecordingInfo(string id, string uri)
+        public RecInfo(string id, string uri)
```

Comment:
What does this really do? We are actually building the initial best practices of how to use recording and I feel RecInfo and the RecordingInfo actually have a deeper meaning. 

What is the use case for RecInfo? What is the use case for RecordingInfo?

Reply:
Local recording class named as LocalRecordingInfo,